src/routes/mission/[...mission]/_DeMiLErrorDialog.svelte

```diff
@@ -0,0 +1,90 @@
+﻿<script lang="ts">
+	import Dialog from '$lib/controls/Dialog.svelte';
+	import type { RepoModule } from '$lib/repo';
+
+	export let demilHelpState:
+		| 'Error'
+		| 'InvalidVersion'
+		| 'NotInstalled'
+		| 'MissingModules'
+		| 'MissionNotFound'
+		| undefined;
+	export let steamID: string;
+	export let missingModules: RepoModule[];
+	export let demilErrorMessage: string;
+	export let demilVersion: string | undefined;
+
+	let dialog: HTMLDialogElement;
+
+	$: {
+		if (dialog !== undefined) {
+			if (demilHelpState !== undefined) {
+				dialog.showModal();
+			} else {
+				dialog.close();
+			}
+		}
+	}
+</script>
+
+<Dialog bind:dialog>
+	{#if typeof demilHelpState !== 'undefined'}
+		<div class="block">
```

Comment:
Can you remove this block element? It makes it look the dialog has a weird thick gray border around it.